conda/recipes/cusignal/meta.yaml

```diff
@@ -9,7 +9,7 @@ package:
   version: {{ version }}
 
 source:
-  path: ../../..
+  git_url: ../../..
```

Comment:
This change is required to prevent this issue:
```
AssertionError: Can't merge/copy source into subdirectory of itself.  Please create separate spaces for these things.
  src: /workspace
  dst: /workspace/.conda-bld/work
```